Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -132,6 +132,19 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
       meta = (ClampMin = 0))
   int32 MaximumSimultaneousTileLoads = 20;
 
+  /**
+   * @brief The maximum number of bytes that may be cached.
+   *
+   * Note that this value, even if 0, will never
+   * cause tiles that are needed for rendering to be unloaded. However, if the
+   * total number of loaded bytes is greater than this value, tiles will be
+   * unloaded until the total is under this number or until only required tiles
+   * remain, whichever comes first.
+   */
+
```

Comment:
I'm not sure if Doxygen will care about it, but please remove the blank line for consistency with the other properties.

Reply:
Fixed ! 👍